src/lib/modes/cipher_mode.h

```diff
@@ -84,6 +84,16 @@ class BOTAN_PUBLIC_API(2, 0) Cipher_Mode : public SymmetricAlgorithm {
    public:
       /**
       * Begin processing a message with a fresh nonce.
+      * Start processing a message with a fresh nonce.
+      *
```

Comment:
One should be enough...

```suggestion
      /**
      * Begin processing a message with a fresh nonce.
      *
```